hypermedia-pipeline/src/defaults/html.pipe.js

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2018 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+const pipeline = require('../../index.js');
+const { before, after, log } = require('./default.js');
+
+const fetch = require('../html/fetch-markdown.js');
+const parse = require('../html/parse-markdown.js');
+const meta = require('../html/get-metadata.js');
+const html = require('../html/make-html.js');
+const responsive = require('../html/responsify-images.js');
+const emit = require('../html/emit-html.js');
+const type = require('../html/set-content-type.js');
+
+module.exports.pipe = (cont, params, secrets, logger = log) => {
```

Comment:
I would prefer not do have direct functions directly assigned to `module.exports`